juniper_official/Solutions/Network-services-mismatch/online_mpc.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
420
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/Network-services-mismatch/online_mpc.yml
  6:1       error    too many blank lines (1 > 0)  (empty-lines)

Reply:
removed the blank link